fivetran_provider_async/example_dags/example_fivetran_async.py

```diff
@@ -18,18 +19,24 @@
 )
 
 with dag:
-    fivetran_sync_start = FivetranOperator(
-        task_id="fivetran-task",
+    fivetran_async_op = FivetranOperatorAsync(
+        task_id="fivetran_async_op",
         fivetran_conn_id="fivetran_default",
-        connector_id="{{ var.value.connector_id }}",
+        connector_id="bronzing_largely",
     )
 
-    fivetran_sync_wait = FivetranSensorAsync(
-        task_id="fivetran-sensor",
+    fivetran_sync_op = FivetranOperator(
+        task_id="fivetran_sync_op",
         fivetran_conn_id="fivetran_default",
-        connector_id="{{ var.value.connector_id }}",
+        connector_id="bronzing_largely",
+    )
+
+    fivetran_async_sensor = FivetranSensorAsync(
+        task_id="fivetran_async_sensor",
+        fivetran_conn_id="fivetran_default",
```

Comment:
Can't this `fivetran_conn_id` added in default args?

Reply:
Changed in [a6fa2d1](https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/commits/a6fa2d1f3c33b5c90c7f9ae64890b11aa0928dd2)